api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 217**
+
+* [`POST /mobile_push/test_notification`](/api/test-notify): Added new endpoint
+  to send a test push notification to a mobile device or devices.
```

Comment:
I'm not adding a link to this in `rest-endpoints.md`, because that makes `zerver/openapi/test_curl_examples.py` try to validate the `zulip.yaml` example, but I don't think we can feasibly do that - since successful operation of this endpoint requires talking to the bouncer

Reply:
Yeah that sounds reasonable for now; @laurynmm FYI, we should probably discuss in "api documentation", but probably we want to just add some additional filtering to that test.